### PR TITLE
Example pr

### PR DESCRIPTION
**Pull request by todorus**
dude

### CODE REVIEW

File: shopping_list.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+1. Bakery and Bread
+2. Meat and Seafood
```
Comment (todorus) [inline shopping_list.md:2]: Do we really need this?

File: shopping_list.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+3. Pasta and Rice
+4. Oils, Sauces, Salad Dressings, and Condiments
+5. Cereals and Breakfast Foods
+6. Soups and Canned Goods
+7. Frozen Foods
```
Comment (Tilana) [inline shopping_list.md:7]: That's not healthy... let's go for fresh food

File: shopping_list.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+8. Dairy, Cheese, and Eggs
+9. Snacks and Crackers
+10. Produce
+11. Drinks
```
Comment (Tilana) [inline shopping_list.md:11]: Alcohol?

File: testing.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ornare leo dui, vitae aliquam lectus porttitor non. Fusce molestie vulputate libero quis sagittis. Fusce in odio diam. Donec non luctus urna. Morbi sit amet justo in est vehicula varius vel nec arcu. Duis ultrices tempus ante. Praesent dapibus efficitur sem, semper consequat purus iaculis eu. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae;
+
```
Comment (todorus) [inline testing.md:2]: another one

**Comment by Tilana** [comment]
Can we close this PR and don't accept the changes?

**Pull request closed without merging**